src/components/Uploads/UploadsListItem.js

```diff
@@ -34,14 +32,8 @@ const StyledRightContainer = styled.div`
   flex-direction: column;
   align-items: center;
   justify-content: space-between;
-`;
-
-const StyledFileName = styled.div`
-  color: ${colors.white87};
-`;
-
-const StyledStatus = styled.div`
-  color: ${colors.white60};
+  padding-right: 16px;
+  padding-left: 16px;
```

Comment:
Why do we have 16px left padding here and in for the `<List>` in src/components/Uploads/UploadsList.js ?
Isn't it enough to specify left padding in one place only?

Reply:
Maybe it makes sense, can't see the DOM structure right now because the button doesn't redirect to the uploads page.

Reply:
with padding
![image](https://user-images.githubusercontent.com/60269784/97790384-eee40300-1bd0-11eb-8b1a-49379880ebd1.png)

without padding
![image](https://user-images.githubusercontent.com/60269784/97790403-22269200-1bd1-11eb-9e13-9d8cf9cf5a44.png)

Reply:
ok :)